media-video/motion/files/motion.confd-r4

```diff
@@ -0,0 +1,23 @@
+# Motion OpenRC conf.d file
+
+
+# Set the user and group under which motion will be run
+#MOTION_USER="motion"
+#MOTION_GROUP="motion"
```

Comment:
Why is everything commented in this file? Looks ok to me.

Reply:
All the comments are just documentation, showing default values.